tgui/packages/tgui/styles/interfaces/PersonalCrafting.scss

```
@@ -0,0 +1,26 @@
+@use '../base.scss';
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)